plugins/modules/one_image_info.py

```diff
@@ -307,7 +307,7 @@ def run(self, one, module, result):
             images = self.get_all_images().IMAGE
 
         self.result = {
-            'images': [OpenNebulaModule.get_image_info(image) for image in images]
+            'images': [OpenNebulaModule.get_image_info(self, image) for image in images]
```

Comment:
Since the `self` object is derived from `OpenNebulaModule`, why not:
```suggestion
            'images': [self.OpenNebulaModule.get_image_info(image) for image in images]
```
which is more pythonic and (allegedly) more OO-like as well?

Reply:
Changed it, thanks!

Reply:
TIL some new Python syntax :)